testing/tests/index.test.ts

```diff
@@ -1070,6 +1070,21 @@ test("image flaws with bad images", () => {
         "File not present on disk, an empty file, or not an image"
     ).length
   ).toBe(4);
+  // Check the line and column numbers for html <img> tags in markdown
```

Comment:
It seems that the start column of markdown images are the index of `!` (+1). Only add line and column test for html <img> elements